### DISCUSSION TITLE
Current readme config?

### DISCUSSION THREAD

**Discussion by semi710** [ErikReider/SwayNotificationCenter#418] [Q&A] [answered]
![image](https://github.com/ErikReider/SwayNotificationCenter/assets/60490474/c7204897-ffaf-4b61-a0dd-735fa97546ed)
currently the README showing this image.

@ErikReider where can I get this exact config?

**Comment by ErikReider** [comment, marked as answer]
Should be the default one. Here's my [config](https://github.com/ErikReider/Linux/blob/d9fc807d6a7bdd2a5af00c9e45b602398af6371b/dotfiles/swayConfig/swaync/config.json) just in case it's not the default one

**Reply by semi710** [reply]
i was looking it with the blur

**Reply by ErikReider** [reply]
Oh, that's not swaync. That's swayfx applying the blur. Hyprland also has a similar feature

**Reply by semi710** [reply]
> Oh, that's not swaync. That's swayfx applying the blur. Hyprland also has a similar feature

Ohh now I got it ty mate
